src/packer.js

```diff
@@ -114,6 +114,11 @@ var Packer = new Lang.Class ({
         if ('thumbnail' in metadata)
             metadata['thumbnail'] = 'ekn:///' + this._hashify(metadata['thumbnail']);
 
+        if ('tags' in metadata)
+            metadata['tags'].push('EknSetObject');
+        else
+            metadata['tags'] = ['EknSetObject'];
+
```

Comment:
Not sure this is necessary, I mean, the tools need to make sure to add the "right" tags, and they do:

* https://github.com/endlessm/basin/blob/master/tools/incubator.js#L180
* https://github.com/endlessm/basin/blob/master/tools/processor.js#L99

Reply:
The problem is I actually want to move away from hard-coding e-k-l-specific object models into the core bits of basin (to make it more flexible, for other use cases like recipes). Doing this goes in the opposite direction, that is why I suggested that maybe we need a tool for crating permanent shards for e-k-l sets (would include this, and maybe even downloading of the thumbnails).

Reply:
Following up with this, I definitely need to re-design the basin manifest format itself to make the packer less e-k-l-centrist, so ideally we would not even have a "sets" section anymore, just objects.

If we add this hard-coded ekn specific tag here, I will need you to remove it from the tools so we don't end up adding duplicated tags, like:

```
{
  "childTags": [
    "Sports"
  ],
  "tags": [
    "EknSetObject",
    "EknSetObject"
  ],
  "title": "Sports",
  "featured": false,
  "@id": "ekn:///f99aecef3d12e02dcbb6260bbdd35189c89e6e73",
  "@type": "ekn://_vocab/SetObject"
}
```
And that will indeed move the code farther from what I am envisioning. So, to me it clear that we shouldn't do this, and that we should a) add the tag in the example you wrote for the docs, or b) add another tool that handles this. But a) sounds perfectly reasonable to me.

Reply:
A simpler middle ground could be to just check if its already included, but acknowledging that will this "feature" of adding the tag automatically in the packer will go away.